src/main/java/games/strategy/triplea/ai/proAI/ProTechAI.java

```diff
@@ -243,7 +240,7 @@ private static float getStrengthOfPotentialAttackers(final Territory location, f
               }
             }
             seaStrength += strength(loadedUnits, true, false, transportsFirst);
-            transportsCounted = true;
+            break;
```

Comment:
Not the fault of this PR, but, wow, is this loop a mess.  The `break` is essentially the last statement in the loop started on line 182.  I guess the loop is basically a combination of "find a specific `Territory`" (lines 183-195), and then "do something with that `Territory`" (lines 196-242).

I don't expect you to change this.  Just ranting... :smile:

Reply:
Up to you to fix it in another PR 😄